exercises/sieve/tests/sieve.rs

```diff
@@ -0,0 +1,48 @@
+extern crate sieve;
+
+use sieve::*;
+
+#[test]
+fn limit_lower_than_the_first_prime() {
+    assert_eq!(Sieve::primes_up_to(1), []);
+}
+
+#[test]
+#[ignore]
+fn limit_is_the_first_prime() {
+    assert_eq!(Sieve::primes_up_to(2), [2]);
+}
+
+#[test]
+#[ignore]
+fn primes_up_to_10() {
+    assert_eq!(Sieve::primes_up_to(10), [2,3,5,7]);
```

Comment:
since I noticed you did a `rustfmt` on your code, I suppose I might as well ask - did it say anything about commas in this list? There are no spaces after commas here, but spaces after commas elsewhere. Just curious.